src/clientsettingsdlg.cpp

```diff
@@ -334,6 +334,11 @@ CClientSettingsDlg::CClientSettingsDlg ( CClient*         pNCliP,
         "you will not have much fun using the " ) + APP_NAME +
         tr ( " software." ) + TOOLTIP_COM_END_TEXT );
 
+    QString wtNumMixerPanelRows = "<b>" + tr ( "Number of Mixer Panel Rows" ) + ":</b> " +
```

Comment:
Convention for all other What's this string variables seems to be to use str*.

```suggestion
    QString strNumMixerPanelRows = "<b>" + tr ( "Number of Mixer Panel Rows" ) + ":</b> " +
```

Reply:
Ah, I've no idea what I was thinking..!